fbw-a32nx/src/systems/fmgc/src/guidance/GuidanceController.ts

```diff
@@ -209,6 +211,40 @@ export class GuidanceController {
         }
     }
 
+    private updateEfisData() {
+        const gs = SimVar.GetSimVarValue('GPS GROUND SPEED', 'Knots');
+        const flightPhase = getFlightPhaseManager().phase;
+        const etaComputable = flightPhase >= FmgcFlightPhase.Takeoff && gs > 100;
+        const activeLeg = this.activeGeometry.legs.get(this.activeLegIndex);
```

Comment:
Sorry for being a pain, but just to make sure if `activeGeometry` is null.

```suggestion
        const activeLeg = this.activeGeometry?.legs.get(this.activeLegIndex);
```